rclcpp/include/rclcpp/executable_list.hpp

```diff
@@ -38,7 +38,31 @@ struct ExecutableList
   ExecutableList();
 
   RCLCPP_PUBLIC
-  virtual ~ExecutableList();
+  ~ExecutableList();
```

Comment:
Why remove this from being `virtual`? It should either be `virtual`, non-virtual but class is marked `final`, or non-virtual but destructor is protected.

I'll change this unless someone speaks up...

Reply:
I removed `virtual` because the class is not intended to be derived. The class should be marked `final` as you propose.

Reply:
Ok I'll do that on my pull request.

Reply:
See 6b15351